tests/tensorflow/test_tensorflow2_autolog.py

```diff
@@ -123,18 +127,6 @@ def __getitem__(self, index):
     return DataGenerator()
 
 
-@pytest.fixture
-def clear_tf_keras_imports():
```

Comment:
Remove this harmful fixture that I wrote awhile ago, which turns out to be obviated by `clear_fluent_autologging_import_hooks` and causes esoteric TensorFlow import errors in TF < 2.1.0 (e.g. importing `tensorflow.estimator` fails after this fixture runs)